deployed POM references non-existent parent

`coat-libs` consumers with Maven fail as of 13.0.0, for example `clas12-timeline` cannot consume:
```
Error:  Failed to execute goal on project clas12-timeline: Could not collect dependencies for project org.jlab.clas.timeline:clas12-timeline:jar:3.1.0
Error:  Failed to read artifact descriptor for org.jlab.coat:coat-libs:jar:13.0.0
Error:  	Caused by: The following artifacts could not be resolved: org.jlab.clas:coatjava:pom:13.0.0 (absent): Could not find artifact org.jlab.clas:coatjava:pom:13.0.0 in clas12maven (https://clasweb.jlab.org/clas12maven)
Error:  
Error:  -> [Help 1]
Error:  
Error:  To see the full stack trace of the errors, re-run Maven with the -e switch.
Error:  Re-run Maven using the -X switch to enable full debug logging.
Error:  
Error:  For more information about the errors and possible solutions, please read the following articles:
Error:  [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
Error: Process completed with exit code 1.
```
The deployed POM file is not correct.

--- a/.gitignore
+++ b/.gitignore
@@ -9,6 +9,7 @@ tmp
 # deployment
 /myLocalMvnRepo
 /coatjava-*.tar.gz
+/common-tools/coat-libs/dependency-reduced-pom.xml
 /common-tools/coat-libs/.flattened-pom.xml
 
 *.class

--- a/.gitignore
+++ b/.gitignore
@@ -9,7 +9,7 @@ tmp
 # deployment
 /myLocalMvnRepo
 /coatjava-*.tar.gz
-dependency-reduced-pom.xml
+/common-tools/coat-libs/.flattened-pom.xml
 
 *.class
 *lundfiles*.dat

--- a/common-tools/coat-libs/pom.xml
+++ b/common-tools/coat-libs/pom.xml
@@ -233,14 +233,33 @@
         </configuration>
       </plugin>
 
+      <!-- generate minimal POM file for deployment, containing GAV (groupID, artifactId, version) -->
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>flatten-maven-plugin</artifactId>
+        <version>1.7.0</version>
+        <configuration>
+          <flattenMode>clean</flattenMode>
+        </configuration>
+        <executions>
+          <execution>
+            <id>flatten</id>
+            <phase>package</phase>
+            <goals>
+              <goal>flatten</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
       <!-- deployment configuration -->
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-deploy-plugin</artifactId>
         <version>3.1.4</version>
         <configuration>
-          <!-- Use the reduced POM -->
-          <pomFile>${project.build.directory}/dependency-reduced-pom.xml</pomFile>
+          <!-- Use the minimal GAV POM -->
+          <pomFile>${project.build.directory}/.flattened-pom.xml</pomFile>
         </configuration>
       </plugin>
 

--- a/common-tools/coat-libs/pom.xml
+++ b/common-tools/coat-libs/pom.xml
@@ -185,7 +185,7 @@
           </execution>
         </executions>
         <configuration>
-          <createDependencyReducedPom>true</createDependencyReducedPom>
+          <createDependencyReducedPom>false</createDependencyReducedPom> <!-- minimal POM will be created by 'flatten-maven-plugin' -->
           <artifactSet>
             <excludes>
               <exclude>cnuphys:cnuphys</exclude>
@@ -239,7 +239,7 @@
         <artifactId>flatten-maven-plugin</artifactId>
         <version>1.7.0</version>
         <configuration>
-          <flattenMode>clean</flattenMode>
+          <flattenMode>fatjar</flattenMode>
         </configuration>
         <executions>
           <execution>

